pac4j-core/src/main/java/org/pac4j/core/context/session/SessionStore.java

```diff
@@ -48,16 +51,16 @@
      * Get the native session as a trackable object.
      *
      * @param context the web context
-     * @return the trackable object or <code>null</code> if this is not supported
+     * @return the trackable object
      */
-    Object getTrackableSession(C context);
+    Optional<Object> getTrackableSession(C context);
 
     /**
      * Build a new session store from a trackable session.
      *
-     * @param context the web context
+     * @param context          the web context
      * @param trackableSession the trackable session
-     * @return the new session store or <code>null</code> if this is not supported
+     * @return the new session store
      */
     SessionStore<C> buildFromTrackableSession(C context, Object trackableSession);
```

Comment:
This should be an `Optional<SessionStore<C>>` as a new session store could be built from an optional `trackableSession`.